src/store/reducers/appReducer.js

```diff
@@ -30,13 +30,28 @@ export const epics = createEpicScenario({
     type: 'APP_INITIALIZE',
     epic: () => [
       epics.actions.fetchUser(),
-      epics.actions.fetchAzureMapsKey(),
       epics.actions.fetchDeviceGroups(),
       epics.actions.fetchLogo(),
-      epics.actions.fetchReleaseInformation()
+      epics.actions.fetchReleaseInformation(),
+      epics.actions.fetchSolutionSettings()
     ]
   },
 
+  /** Log diagnostics data */
+  logEvent: {
+    type: 'APP_LOG_EVENT',
+    epic: ({ payload }, store) => {
+      const diagnosticsOptIn = getDiagnosticsOptIn(store.getState());
+      if (diagnosticsOptIn) {
+        return DiagnosticsService.logEvent(payload)
+          .flatMap(_ => Observable.empty())
+          .catch(_ => Observable.empty())
```

Comment:
Why not use handleError(fromAction) like the other epics? If there's a valid reason to be different, add a comment.  We can see *that* its different from the code itself, but we cannot tell *why* its different. Whys are important to document.

Reply:
added comment

Reply:
I don't see a comment.  My original question is about the flatMap and catch in the *if* block?

Reply:
This is not error handling. Wether or not the request succeeds, we don't want any more action to be executed. In that case, the epic must return an empty observable.